packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2298,6 +2298,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         this.dataStores.deleteUnusedRoutes(dataStoreUnusedRoutes);
     }
 
+    public revive(usedRoutes: string[]) {
```

Comment:
Because it's part of the `IGarabgeCollectionRuntime` class and the `ContainerRuntime` class, I'd prefer a more generic name in case we want to revive gc objects that aren't tombstoned. Do you think this is a valid concern?